### DISCUSSION TITLE
Blocco/sblocco  registro

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#137] [Richieste di aiuto] [answered]
Buongiorno, vorrei sapere se è possibile bloccare dopo una determinata ora l'accesso al registro per i docenti .Ex.blocco ore 23.00 ,sblocco ore 8.00

**Comment by trinko** [comment, marked as answer]
Attualmente no. 
I parametri esistenti nella configurazione **blocco_inizio** e **blocco_inizio** erano pensati per gestire il contrario, cioè blocco del login dei docenti durante l'orario di lezione, in modo da obbligarli a usare un'altra modalità gestita direttamente dai dispositivi presenti nelle classi (lettore di impronta digitale).

In ogni caso, penso che per il prossimo anno scolastico inseriremo una qualche gestione di blocco orario per tutte le modalità di accesso.
